app/constants/request_date.py

```diff
@@ -0,0 +1 @@
+RELEASE_PUBLIC_DAYS = 20
```

Comment:
There are a number of standard dates that the system uses that should go in this file. I'm not sure the filename should be request_date, but we can work on that later. Just noting it here so we don't forget.